docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -15,53 +15,63 @@ use the following commands to launch the placement translator; it
 requires the ESPW to be current at start time.
 
 ```shell
-# TODO: placement-translator needs access to multiple configs. Will remove when controllers support spaces.
-kubectl ws root:espw
-placement-translator &
-sleep 10
+(
+  export KUBECONFIG=$SM_CONFIG
+  placement-translator &
+  sleep 10
+)
 ```
 
 The following commands wait for the placement translator to get its
 job done for this example.
 
 ```shell
-# TODO: unfortunately, the $FLORIN_WS and $GUILDER_WS are mailbox ws names that we are not supporting in the space framework yet
+$FLORIN_SPACE_CONFIG="${PWD}/temp-space-config/${FLORIN_SPACE}"
```

Comment:
"FLORIN_SPACE" is a confusing name for a variable that holds the name of the _mailbox space_ for the florin WEC. I think that "FLORIN_MAILBOX_SPACE" would be much clearer.

An analogous comment applies toe "FLORIN_SPACE_CONFIG".